frontend/src/features/rollout-announcement/components/ProgressIndicator.tsx

```diff
@@ -0,0 +1,77 @@
+import { useMemo } from 'react'
+import { Box } from '@chakra-ui/react'
+
+import { MotionBox } from '~components/motion'
+
+const ActiveIndicator = (): JSX.Element => (
+  <Box
+    width="1.5rem"
+    height="0.5rem"
+    borderRadius="full"
+    backgroundColor="secondary.500"
```

Comment:
might want to make this a button too (that does nothing), and add aria-label and focus styles too. these can maybe be in a later PR if designers don't mind

Reply:
Created an [issue](https://github.com/opengovsg/FormSG/issues/3977) to track this, will add in a separate PR